### ISSUE TITLE
Hidden field input attributes

### ISSUE THREAD

**Issue by shoored** [verbb/formie#321]
**What are you trying to do?**
I'm trying to add an attribute to a hidden field input.

**What's your proposed solution?**
Verbb knows best.

When adding a hidden field, it would be nice to be able to add attributes to the input as well. Currently it's only possible to add attributes to the field container. Single Text fields for instance do have this option.

For example, this would allow for easy mapping of Alpine JS data to hidden input fields, where data is collected outside the Formie form.

**Comment by engram-design** [comment]
Not _quite_ sure why these were missing! Should be fixed in the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.13",
  "...": "..."
}
```

Then run `composer update`.

**Comment by shoored** [comment]
Amazing, thanks!

**Comment by engram-design** [comment]
Fixed in [1.3.14](https://github.com/verbb/formie/releases/tag/1.3.14)
